sktime/forecasting/trend/_polynomial_trend_forecaster.py

```diff
@@ -151,6 +184,44 @@ def _predict(self, fh=None, X=None):
         y_pred.name = self._y.name
         return y_pred
 
+    def _predict_var(self, fh=None, X=None, cov=False):
+        """Compute the variance at each forecast horizon."""
+        if self.prediction_intervals is False:
+            raise ValueError(
+                "Prediction intervals were not calculated during fit. \
+                Set prediction_intervals=True at initialization."
+            )
+
+        # 1. get X (design matrix) and M = (X^t X)^-1
+        t_train = _get_X_numpy_int_from_pandas(self.train_index_).flatten()
+        X = np.polynomial.polynomial.polyvander(t_train, self.degree)
+        if not self.get_params()["with_intercept"]:
```

Comment:
Same clarification as above, possibly it can be just self.with_intercept. Please let me know if I'm missing something.